src/pudl/metadata/resources/eia861.py

```diff
@@ -642,6 +786,7 @@
         "etl_group": "eia861",
     },
     "core_eia861__yearly_utility_data_rto": {
```

Comment:
`core_eia861__yearly_utility_rtos`? `core_eia861__yearly_utility_iso_rto`?

Reply:
To be addressed in #3301